queries.sql

```diff
@@ -12,7 +12,46 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 
 SELECT * FROM animals WHERE neutered = true;
 
-SELECT * FROM ANIMALS WHERE name NOT LIKE '%Gabumon%';
+SELECT * FROM animals WHERE name NOT LIKE '%Gabumon%';
 
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
+UPDATE animals SET species = 'unspecified' WHERE species IS NULL;
+
+ROLLBACK;
+
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+
+COMMIT;
```

Comment:
- [ ]  Please kindly always start the transaction with `BEGIN;` 😉
- [ ] Please select species from animals after the update to verify if the change was made 🙃
- [ ] Also kindly select species from animals after the commit to verify if the change was made 🙃